apps/artemis-web/components/bgp-table/bgp-table.tsx

```diff
@@ -670,16 +669,17 @@ const BGPTableComponent = (props) => {
     setPage(page);
     setSizePerPage(sizePerPage);
 
-    if (currentIndex && sizePerPage) {
-      setOffsetState(currentIndex);
-      setLimitState(sizePerPage);
-    }
+    console.log(currentIndex)
```

Comment:
just making a note that these console logs should be removed when the PR is in a "ready" state